client/state/comments/reducer.js

```diff
@@ -190,15 +196,24 @@ export const totalCommentsCount = createReducer(
 export const errors = createReducer(
 	{},
 	{
-		[ COMMENTS_ERROR ]: ( state, action ) => ( {
-			...state,
-			[ `${ action.siteId }-${ action.commentId }` ]: { error: true },
-		} ),
+		[ COMMENTS_ERROR ]: ( state, action ) => {
```

Comment:
how would we unset this? say the network comes back up or it was a transient HTTP 500...